app/src/main/java/uk/co/ribot/androidboilerplate/data/SyncService.java

```diff
@@ -60,7 +60,7 @@ public void onCompleted() {
 
                     @Override
                     public void onError(Throwable e) {
-                        Timber.w(e, "Error syncing.");
```

Comment:
The correct way is with the Throwable as the first argument. https://github.com/JakeWharton/timber/blob/master/timber/src/main/java/timber/log/Timber.java line 53